### ISSUE TITLE
False-positive for indentation

### ISSUE THREAD

**Issue by boris-petrov** [clutchski/coffeelint#597]
```coffeescript
class Foo
  sum: (a, b, \
      c) ->
    a + b + c
```

This here gives an incorrect indentation on line 4 (the last one). I think this is a bug.
